tests/integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -92,6 +92,7 @@ else
 	)
 
 	K8S_TEST_NORMAL_HOST_UNION=( \
+		"k8s-debug-console.bats" \
```

Comment:
I think it should run fine on the small union. Or do it need to run on VMs with more resources (e.g. more vCPUs)?

Reply:
I've moved it to the small union. Wasn't quite sure which one to add it to based on the comments and notes, so thanks for the clarification here.